.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  release:  
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
+    name: Install and test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade tox pytest coverage coveralls
+      - name: Install imaplib2
+        run: python setup.py install
```

Comment:
You don't need this, tox should install the package for you

Reply:
Done.